SmartVision/WindowsForms/FaceDetection.cs

```diff
@@ -41,10 +44,32 @@ private void EnableWebcam()
         /// </summary>
         private void ProcessFrame(object sender, EventArgs e)
```

Comment:
Since this is a Form class, it should not contain the **EnableWebcam(), ProcessFrame()** and **FaceDetectionFromFrame()** method implementations.
They should be moved to respective classes and called upon from the Form class when needed.